### PR TITLE
BGDIINF_SB-3114: Created PPBGDI package

### PR DESCRIPTION
**Pull request by ltflb-bgdi**
## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

-

### CODE REVIEW

File: packages/ppbgdi/data_stream/python_logs/fields/fields.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          description: The easting orientation of a tile
+        - name: row
+          type: long
+          description: The northing orientation of the tile
+        - name: zoom
+          type: long
+          description: The zoom level of the tile
+        - name: time
+          type: keyword
+          description: The publication time of the resource
```
Comment (boecklic) [inline packages/ppbgdi/data_stream/python_logs/fields/fields.yml:25]: if possible, these fields would ideally go into a `entity-layer.json` or sth...

File: packages/ppbgdi/data_stream/python_logs/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      field:
+        - first_char
+  - set:
+      field: event.outcome
+      value: failure
+      if: 'ctx?.http?.response?.status_code != null && ctx?.http?.response?.status_code >= 400'
+  - set:
+      field: event.outcome
+      value: failure
+      if: "ctx?.event?.severity >= 40"
```
Comment (boecklic) [inline packages/ppbgdi/data_stream/python_logs/elasticsearch/ingest_pipeline/default.yml:44]: - `status_code` should dominate `severity`
- failure for `status_code >= 500`

File: packages/ppbgdi/data_stream/python_logs/fields/fields.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - name: system
+      type: keyword
+      description: The BGDI system
+    - name: crs
```
[the remark this answers is not in the document]
Comment (ltflb-bgdi) [reply packages/ppbgdi/data_stream/python_logs/fields/fields.yml]: Done

File: packages/ppbgdi/data_stream/python_logs/fields/fields.yml (hunk 1 above)
Comment (ltflb-bgdi) [reply packages/ppbgdi/data_stream/python_logs/fields/fields.yml]: Done.

File: packages/ppbgdi/data_stream/python_logs/elasticsearch/ingest_pipeline/default.yml (hunk 2 above)
Comment (ltflb-bgdi) [reply packages/ppbgdi/data_stream/python_logs/elasticsearch/ingest_pipeline/default.yml]: Done

**Comment by ltflb-bgdi** [comment]
Replaced by https://github.com/geoadmin/infra-elastic-integrations/pull/4

**Pull request closed without merging**